data/tosca/yorc-google-types.yml

```diff
@@ -8,6 +8,10 @@ metadata:
 imports:
   - yorc: <yorc-types.yml>
 
+artifact_types:
+  yorc.artifacts.google.Deployment.Address:
```

Comment:
Just a question, I do not have a definitive position on  that but should we have a specific artifact type for each node type or a generic one like `yorc.artifacts.google.Deployment` used for all gcp resources ?

I think I prefer the second option as we will probably have some other nodes like this in future.
Do you see any drawbacks with the second option?
@laurentganne @adidanes WDYT?

Reply:
Ok for the second option in my opinion